src/schnetpack/nn/nuclear_embedding.py

```diff
@@ -0,0 +1,87 @@
+import math
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from schnetpack.electron_configurations import electron_config
+
+
+class NuclearEmbedding(nn.Module):
+    """
+    Embedding which maps scalar nuclear charges Z to vectors in a
+    (num_features)-dimensional feature space. The embedding consists of a freely
+    learnable parameter matrix [Zmax, num_features] and a learned linear mapping
+    from the electron configuration to a (num_features)-dimensional vector. The
+    latter part encourages alchemically meaningful representations without
+    restricting the expressivity of learned embeddings.
+    Using complexe nuclear embedding can have negative impact on the model performance, when spin charge embedding is activated
+
+    Arguments:
+        num_features (int):
+            Dimensions of feature space.
+        Zmax (int):
+            Maximum nuclear charge +1 of atoms. The default is 87, so all
+            elements up to Rn (Z=86) are supported. Can be kept at the default
+            value (has minimal memory impact).
+    """
+
+    def __init__(
+        self, num_features: int, Zmax: int = 87, zero_init: bool = True
```

Comment:
change order of `num_features` and `Zmax` to be equal to the order of arguments in `nn.Embedding`

Reply:
done in c465ce4